scoped/sign-up/sign-up.md

```diff
@@ -0,0 +1,42 @@
+**Name**
+Signup
+
+----
+
+**Actor**  
+Anonymous user
+
+----
+ 
+**Subject area**  
+Any
+
+----
+
+**Preconditions**  
+Hit the signup button from the nav bar or tried to perform an authenticated action, was presented with the log in or sign up modal and clicked sign up
+
+----
+
+**Termination outcome**  
+User's account is created and redirected to log in page
+
+----
+
+**Basic flow**  
+
+- User presented with short form and explanation summary
+- User briefly reads paragraph, enters display name, email and password
+- Any appropriate prompts and error messages are displayed underneath the fields
+- User clicks sign up, once successful user is redirected to login page
```

Comment:
We need to account for the "has account on another site on the network, wants to add this site" case.  That can be a separate use case (linked to this one); let's not lose track of it.